.changeset/little-pianos-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomiclabs/hardhat-vyper": patch
+---
+
+Fixed escaped double quote in vyper docs.
```

Comment:
Doc changes don't need changesets because there is nothing to be released. The website is automatically updated with each change in `main`.